### PR TITLE
updated home-design scss

### PR DESCRIPTION
**Pull request by Satyam1RP**
@Lalit-gour1 Here is the updated scss for home-design component.

### CODE REVIEW

**Comment by Lalit-gour1** [comment]
@Satyam1RP  Please resolve conflict

**Comment by Satyam1RP** [comment]
@Lalit-gour1  conflicts resolved.

**Comment by mathrik** [comment]
You have new conflicts, but it looks good otherwise.

File: Assets/js/main.js [hunk 1]
```diff
@@ -3,13 +3,11 @@ import bannerCarouselComponent from './banner-carousel.js';
...
 
 const app = new App();
 
 const bannerCarouselSlider = bannerCarouselComponent();
 
 const plasticBannerVideoComp = bannerVideoComponent();
 const newFnComp = newFn();
-const headerComponent = headerFn();
-const successStoriesComponent = successStoriesFn();
+const headerComponent = headerFn();
```
Comment (Lalit-gour1) [inline Assets/js/main.js:13]: @Satyam1RP You've removed success stories reference from this file, any reason for that.
Cc:-  @jvaishnavrp

Comment (Satyam1RP) [reply Assets/js/main.js]: @Lalit-gour1 This happened unintentionally at the time of file selection, I fixed and pushed this code.
CC: @jvaishnavrp

**Comment by mathrik** [comment]
New conflicts. I think we must have a some files in version control that we don't want to keep there.

**Pull request closed without merging**